### ISSUE TITLE
Multiverse core is not working properly

### ISSUE THREAD

**Issue by Kannuki-san** [IzzelAliz/Arclight#1403]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

arclight-1.21-1.0.0-SNAPSHOT-b3349e9

### OS & Java versions

OS:windows11 java:openjdk-22

### Plugins and Mods

```raw
multiverse-core-4.3.12
```

### Description

When executing mvtp and moving to a new world, the world is not loaded correctly and the user hits the terrain of the default world, which is not visible on the screen.
Also, when executing commands in this state, the client outputs "Chat disabled in client options" and the command cannot be executed.
If you then log out, you will get a timeout and not be able to log back in.

Similar problems occurred in PhantomWorlds, MyWorlds, and Universes.

### Step to reproduce

1,Install multiverse-core
2,run server
3,Join the Game
4,Execute the command "/mv create test normal"
5,Execute the command "/mvtp test"

### Logs

[latest.log](https://github.com/user-attachments/files/16070379/latest.log)

**Comment by luoxiawuchen** [comment]
Indeed, this should not only be a compatibility issue with MV plugins, but also with related world management plugins

**Comment by Arctrometa** [comment]
+1 I'm facing the same exact issue.

**Comment by Arctrometa** [comment]
A little summary of what I need to do to run into issues with Multiverse Core.
So basically I'm using Multiverse-core to setup two extra worlds in my server and whenever I do /mvtp to any other world I get stuck at the spawn point and the chat gets disabled automatically, in the attached video im using vanilla client to rule out any client side issues. log attached below.
log -https://mclo.gs/FK2uEOA
Video - https://youtu.be/Z10XeOFMrgM

**Comment by qyl27** [comment]
It may fixed by 5bf8aaf, try our newest version.

**Comment by luoxiawuchen** [comment]
> It may fixed by [5bf8aaf](https://github.com/IzzelAliz/Arclight/commit/5bf8aaf889d2d7b20eae433d3bfa55b58fa16c3e), try our newest version.

您好，经过最新构建版本的测试，仍存在一些问题，
一、通过命令传送到另一世界时（包括传送回默认的世界）：
 1、无法破坏方块；
 2、死亡后无法复活，无法点击重生按钮（或点击后无效）
 3、无法执行命令或发送消息至聊天栏，提示：聊天在客户端选项中被禁用。
 4、断开链接后，无法再进入服务器，提示连接超时（应该是一切交互都不响应了）
 5、重启服务器后，首次进入，可正常进行各类交互（只要不切换世界）
![image](https://github.com/user-attachments/assets/3013d745-164a-4d49-85ad-9548a2d51a11)

日志：
https://paste.ubuntu.com/p/WXCYHQZwPk/

**Comment by G3kid0** [comment]
Same issue with the last version.

arclight-fabric-1.21-1.0.0-SNAPSHOT
multiverse-core-4.3.13

**Comment by qyl27** [comment]
cc @IzzelAliz

**Comment by itsjustnickdev** [comment]
^

**Comment by qyl27** [comment]
Try 97e6107?

**Comment by InitAuther97** [comment]
Tested in latest build with Mv 4.3.13, currently operational.
